dataikuapi/dss/ml.py

```diff
@@ -251,20 +251,22 @@ def get_algorithm_settings(self, algorithm_name):
         :return: An AlgortihmSettings (extended dict) for an algorithm.
         :rtype: AlgorithmSettings
```

Comment:
So return type may also be `dict` (also: typo)

```suggestion
        :return: An AlgorithmSettings (extended dict) for a single built-in algorithm,
                 or a plain dict for the settings of custom inline-code (CUSTOM_*) or plugin (PLUGIN_*) algorithms.
        :rtype: AlgorithmSettings | dict
```

Wondering whether we _also_ want to specify those in the docstring, or if it just makes it more complicated.

Reply:
I think that `"CUSTOM_*"` and `"PLUGIN_*"` are not really 1st-class arguments of this method, so I'd rather keep the documentation as clear as possible for the vast majority of usages and be slightly non-exhaustive in the docstring explanation.

Reply:
Marking as resolved as custom and plugin algorithms will be handled separately.